frontend/www/js/omegaup/components/course/Edit.vue

```diff
@@ -254,7 +265,7 @@
 </template>
 
 <script lang="ts">
-import { Vue, Component, Prop, Watch, Ref } from 'vue-property-decorator';
+import { Vue, Component, Prop, Watch, Ref, Emit } from 'vue-property-decorator';
```

Comment:
```suggestion
import { Vue, Component, Prop, Watch, Ref } from 'vue-property-decorator';
```

esto parece que no se está usando.

Reply:
Listo